qa/tasks/cephfs/mount.py

```diff
@@ -233,20 +233,20 @@ def open_background(self, basename="background_file", write=True):
             pyscript = dedent("""
                 import time
 
-                f = open("{path}", 'w')
-                f.write('content')
-                f.flush()
-                f.write('content2')
-                while True:
-                    time.sleep(1)
+                with open("{path}", 'w') as f:
```

Comment:
Just to be clear, you're creating a new file `./{path}` here and you're intentionally not using f-strings?

Reply:
@sebastian-philipp i think it's nice to use `f"{path}"` here. but it's not in the scope of this change.

Reply:
`with open("{path}", 'w') as f:` just looks like a bug in my eyes. As if someone just missed to actually make use of `{path}`. either as an f-string of add the missing `.format(path=path)`